app/models/user.rb

```diff
@@ -1,4 +1,11 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+
+  before_validation :set_default_posts_counter, on: :create
+
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] You have done a good job @baqar-abbas 👍, however, make sure your application asks for confirmation of email as it is [required](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Ask%20for%20confirmation%20of%20email.). we should have something like this:

```ruby
devise :database_authenticatable, :registerable,
		:recoverable, :rememberable, :validatable, :confirmable
```

Kindly, add `:confirmable` 💪.

Reply:
suggested change by the code reviewer has been implemented. Screen shot attached below for your kind reference.

![image](https://github.com/baqar-abbas/blog-app-rails/assets/117091923/2d489fd8-01dd-431c-80fc-c31c749f6aca)

Reply:
Kindly, make sure everything running well 💪.